CHANGELOG.md

```diff
@@ -15,6 +19,13 @@ Canonical reference for changes, improvements, and bugfixes for Boundary.
   [PR2](https://github.com/hashicorp/boundary/pull/2387))
 * workers: Return the worker Boundary binary version on worker list and read
   ([PR](https://github.com/hashicorp/boundary/pull/2377))
+* workers: Addition of worker graceful shutdown ([PR](https://github.com/hashicorp/boundary/pull/2455))
+* credentials: Support json credentials ([PR](https://github.com/hashicorp/boundary/pull/2423))
+* scheduler: Log details when starting scheduler ([PR](https://github.com/hashicorp/boundary/pull/2459))
+* scheduler: Set intervals from config ([PR](https://github.com/hashicorp/boundary/pull/2443))
+* cli(ssh): Do not print private key if consumed
+  ([PR](https://github.com/hashicorp/boundary/pull/2451))
+* refactor: Eliminate gorm references in favor of go-dbw ([PR](https://github.com/hashicorp/boundary/pull/2386))
```

Comment:
This isn't customer facing, refactors are hardly ever added to the changelog

Reply:
ty!